README.md

```diff
@@ -73,27 +118,47 @@ or
 make install
 ```
 
-### Tests
-See the [contribution guide](CONTRIBUTING.md).
-
 # Configuration
 
-`lab` needs your GitLab information in order to interact with to your GitLab
-instance. There are several ways to provide this information to `lab`:
+_lab_ needs your GitLab information in order to interact with to your GitLab instance. There are several ways to
+provide this information to `lab`:
+
+## CI environment variables
```

Comment:
Other parts of the README, such as installation, used triple `#` to sub-items, IMHO using tree `#` is more readable, as is makes and clearer distinction for topics and subtopics.
so 
```suggestion
### CI environment variables
```

Reply:
Fixed.